packages/components/button/stories/button.happo.js

```diff
@@ -0,0 +1,17 @@
+// import { storiesOf } from '@open-wc/demoing-storybook';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)